tests/common.go

```diff
@@ -7398,3 +7489,98 @@ func deleteNamespaces(namespaces []string) error {
 	}
 	return nil
 }
+
+// CreateCloudCredentialNew creates cloud credentials
+func CreateCloudCredentialNew(provider string, ctx context1.Context, orgID string, credName string, uuid string, kubeconfig string) error {
```

Comment:
How about?

` CreateCloudCredential(provider, credName string, uid, orgID string, ctx context1.Context, kubeconfig ...string) error`, and then use `kubeconfig[0]`.

While this approach has its downsides, as we cannot have more than one optional parameter unless we use interfaces, it will nonetheless help maintain the code. Please suggest @mkoppal-px

Reply:
1)CreateCloudCredential will be eventually replaced by CreateCloudCredentialNew (Will use the old name CreateCloudCredential only)
2)Since for RKE we need kubeconfig for each cluster separately, we cannot pass kubeconfig[0]
3)To not use separate method for creating cloud credentials for backup loc and cluster, "kubeconfig" parameter is mandatory